cmd/tracee-rules/main.go

```diff
@@ -174,6 +183,11 @@ func main() {
 				Name:  "list-events",
 				Usage: "print a list of events that currently loaded signatures require",
 			},
+			&cli.IntFlag{
+				Name:  signatureBufferFlag,
+				Usage: "size of the event channel's buffer consumed by signatures",
+				Value: 1000,
```

Comment:
Do we want to enable buffered channels by default? Setting the default value to 0 would fallback to the existing ubbuffered channels.

Reply:
I think having buffer channels should be the default IMO . From a business logic point of view, it doesn't matter wether we are using buffer/unbuffer. However, buffered channel are more efficient as in case of high rate of writing operation to the channel, the logics **won't** be blocked.

Reply:
Yup. That makes sense to me as well. I just wanted to confirm that it's intentional ;)